src/components/QuickOpenModal.js

```diff
@@ -286,6 +290,40 @@ export class QuickOpenModal extends Component<Props, State> {
     return results && results.length ? results.length : 0;
   };
 
+  fuzzyhighlight = (query, results) => {
```

Comment:
Something like this maybe... I haven't had a chance to test so consider this a sketch. 😄 

```jsx
return results.map(result => {
  if (MODIFIERS.includes(query[0]) {
    return result;
  }

  const matching = new Set(...query.split(), ...result.title.split());
  return {
    ...result,
    title: result.title
      .split()
      .map(letter => matching.includes(letter) ? <span className="matched">letter</span> : letter)
  };
});
```

MODIFIERS can be found in utils/quick-open (but you need to export it)

One idea is instead of a span around each letter you could map the ranges and put spans just around the characters that you match.

```
// query = food
<span>foo</span>bar
```